tests/models/gemma2/test_modeling_gemma2.py

```diff
@@ -59,7 +60,7 @@ class Gemma2ModelTest(GemmaModelTest, unittest.TestCase):
         if is_torch_available()
         else ()
     )
-    all_generative_model_classes = ()
+    all_generative_model_classes = (Gemma2ForCausalLM,) if is_torch_available() else ()
```

Comment:
😱 good spot!

Reply:
This was removed because it was faiiling too many tests

Reply:
yes, I skipped those that shouldn't be triggered due to model-specific cache and fixed other failing ones